test/dummy-solver.jl

```diff
@@ -25,7 +25,11 @@ function dummy(
   yzls(x) = begin
     A = m > 0 ? jac(nlp, x) : zeros(T, 0, n)
     @info("", eltype(A))
-    yz = [A' -Pℓ Pu] \ -grad(nlp, x)
+    yz = try
+      [A' -Pℓ Pu] \ -grad(nlp, x)
+    catch
+      zeros(m+nℓ+nu)
```

Comment:
```suggestion
      zeros(m + nℓ + nu)
```

Reply:
I removed the modifications in dummysolver.jl following our discussion. Thanks !